plugins/quay/config.d.ts

```diff
@@ -2,9 +2,14 @@ export interface Config {
   /** Configurations for the Quay plugin */
   quay?: {
     /**
-     * The base url of the Quay instance.
+     * The proxy path for the Quay instance.
      * @visibility frontend
      */
     proxyPath?: string;
+    /**
+     * The base url of the Quay instance.
+     * @visibility frontend
+     */
+    baseUrl?: string;
```

Comment:
Should this be rather named `uiUrl` or something? So it's obvious this value is not used for in API queries at all, only for display and link purposes.

Reply:
Yes would make more sense for the user. Will update this.